pkg/asset/machines/master.go

```diff
@@ -226,12 +225,6 @@ func (m *Master) Load(f asset.FileFetcher) (found bool, err error) {
 		return true, err
 	}
 
-	fileList, err := f.FetchByPattern(filepath.Join(directory, masterMachineFileNamePattern))
-	if err != nil {
-		return true, err
-	}
-
-	m.MachineFiles = fileList
```

Comment:
This will break loading the asset from disk. The `MachineFiles` will be empty when the asset is loaded from disk.